### PR TITLE
fix(ODBC) fix iptc subjects, wrap body text

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

File: server/apps/publish/formatters/aap_ipnews_formatter.py [hunk 1]
```diff
@@ -30,26 +31,47 @@ def format(self, article, output_channel, selector_codes):
...
+                odbc_item['subject_detail'] = ''
 
             odbc_item['take_key'] = article.get('anpa_take_key', None)  # @take_key
             odbc_item['usn'] = article.get('unique_id', None)  # @usn
             if article['type'] == 'preformatted':
                 odbc_item['article_text'] = article.get('body_html', '').replace('\'', '\'\'')  # @article_text
             elif article['type'] == 'text':
-                text = BeautifulSoup(article.get('body_html', '')).text
+                soup = BeautifulSoup(article.get('body_html', ''))
+                text = ''
```
Comment (plamut) [inline server/apps/publish/formatters/aap_ipnews_formatter.py]: Better to use a string builder (i.e. `io.StringIO`) or an accumulator list to build a string from many smaller fragments.

**Comment by plamut** [comment]
:+1:

File: server/apps/publish/formatters/aap_ipnews_formatter_test.py [hunk 2]
```diff
@@ -60,3 +59,29 @@ def TestIPNewsFormatter(self):
...
+                         <p>abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi abcdefghi more</p>',
+            'word_count': '1',
+            'priority': '1'
+        }
+
+        with self.app.app_context():
+            output_channel = self.app.data.find('output_channels', None, None)[0]
+            f = AAPIpNewsFormatter()
+            seq, item = f.format(article, output_channel, self.sel_codes)
+            print(item['article_text'])
```
Comment (mugurrus) [inline server/apps/publish/formatters/aap_ipnews_formatter_test.py]: I guess you can remove print :)

**Comment by mugurrus** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**